### PR TITLE
Address code ql warning

### PR DESCRIPTION
**Pull request by jlevin**

### CODE REVIEW

File: polliwog/polyline/_try_inflection_points.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
                 subdivide_by_length=0.001,
             )
             result_points.append(result)
-        except ValueError:
+        except ValueError("No point of max acceleration found"):
```
[the remark this answers is not in the document]
Comment (paulmelnikow) [reply polliwog/polyline/_try_inflection_points.py:55]: I don't think you can pass exception instances to `except`, only types or a tuple of types.

File: polliwog/polyline/_try_inflection_points.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def main():
             )
             result_points.append(result)
         except ValueError:
-            pass
+            return None
```
Comment (paulmelnikow) [inline polliwog/polyline/_try_inflection_points.py:56]: Satisfying the tooling is not really a sufficient reason to make a change – it needs to make sense independently. Could you explain it?

Comment (jlevin) [reply polliwog/polyline/_try_inflection_points.py]: True... how about printing out a comment before passing, is that reasonable here?

Comment (jlevin) [reply polliwog/polyline/_try_inflection_points.py]: @paulmelnikow

File: polliwog/polyline/_try_inflection_points.py [hunk 3]
```diff
@@ -53,7 +53,7 @@ def main():
             )
             result_points.append(result)
         except ValueError:
-            pass
+            print("no point of max acceleration found.")
```
Comment (paulmelnikow) [inline polliwog/polyline/_try_inflection_points.py:56]: The behavior is fine as it is – there is no need to print anything. I think what the tooling wants is a comment to explain the `pass`.

```suggestion
            # No-op if this point doesn't exist.
            pass
```

**Review by paulmelnikow — APPROVED** [review]

**Pull request merged**